app/src/main-process/menu/build-default-menu.ts

```diff
@@ -322,6 +326,16 @@ export function buildDefaultMenu({
         click: emit('open-external-editor'),
       },
       separator,
+      {
+        id: 'create-issue-in-repository-on-github',
+        label: __DARWIN__
+          ? 'Create Issue on GitHub'
+          : 'Create &Issue on GitHub',
```

Comment:
I don't believe `issue` should be title cased on Windows.
```suggestion
          : 'Create &issue on GitHub',
```